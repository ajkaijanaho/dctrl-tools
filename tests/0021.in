Foo: Ã¤iti
Bar: 42
