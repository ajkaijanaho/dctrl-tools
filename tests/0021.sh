#!/bin/sh

set -e

LC_ALL=C.UTF-8 $TBL_DCTRL 
