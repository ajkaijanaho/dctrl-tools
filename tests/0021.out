+======+=====+
| Foo  | Bar |
+------+-----+
| Ã¤iti | 42  |
+======+=====+
